backend/modules/vector_db/weaviate.py

```diff
@@ -92,40 +157,13 @@ def list_documents_in_collection(
         """
         List all documents in a collection
         """
-        # https://weaviate.io/developers/weaviate/search/aggregate#retrieve-groupedby-properties
-        response = (
-            self.weaviate_client.query.aggregate(collection_name.capitalize())
-            .with_group_by_filter([f"{DATA_POINT_FQN_METADATA_KEY}"])
-            .with_fields("groupedBy { value }")
-            .do()
-        )
-        groups: List[Dict[Any, Any]] = (
-            response.get("data", {})
-            .get("Aggregate", {})
-            .get(collection_name.capitalize(), [])
-        )
-        document_ids = set()
-        for group in groups:
-            # TODO (chiragjn): Revisit this, we should not be letting `value` be empty
-            document_ids.add(group.get("groupedBy", {}).get("value", "") or "")
-        return list(document_ids)
+        pass
```

Comment:
This function can be removed as it is not in the base class.

Reply:
done